test/system/002-setup-admin_spec.js

```diff
@@ -8,47 +11,32 @@ describe('Default Admin User', function () {
     // inbox - a local transport we can use to capture email without an SMTP server
     const inbox = new LocalTransport()
 
-    before(async function () {
```

Comment:
Why have you removed the `before` function? this is needed to reset the test environment.

Reply:
It is because I'm creating flowfuse inside each test method with different configs. I could not find another way.

Reply:
If you need to totally reset the environment between each test then the `before()` and `after()` functions should be renamed to `beforeEach()` and `afterEach()`

Reply:
OK, this can probably stay as it is, but the await in the `after()` will be needed